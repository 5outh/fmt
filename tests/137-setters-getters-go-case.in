<?php
//passes:OrderUseClauses,AddMissingCurlyBraces,SmartLnAfterCurlyOpen,NormalizeLnAndLtrimLines,MergeParenCloseWithCurlyOpen,MergeCurlyCloseAndDoWhile,MergeDoubleArrowAndArray,ExtraCommaInArray,ResizeSpaces,Reindent,ReindentColonBlocks,ReindentLoopColonBlocks,ReindentIfColonBlocks,ReindentObjOps,EliminateDuplicatedEmptyLines,AlignEquals,AlignDoubleArrow,LeftAlignComment,RTrim,SettersAndGettersPass|golang,ConstructorPass|golang,OrderMethod,NormalizeLnAndLtrimLines,Reindent


class A {
	private $a;
	public $b;
	protected $c;
}
