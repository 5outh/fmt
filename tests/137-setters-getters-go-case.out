<?php
//passes:OrderUseClauses,AddMissingCurlyBraces,SmartLnAfterCurlyOpen,NormalizeLnAndLtrimLines,MergeParenCloseWithCurlyOpen,MergeCurlyCloseAndDoWhile,MergeDoubleArrowAndArray,ExtraCommaInArray,ResizeSpaces,Reindent,ReindentColonBlocks,ReindentLoopColonBlocks,ReindentIfColonBlocks,ReindentObjOps,EliminateDuplicatedEmptyLines,AlignEquals,AlignDoubleArrow,LeftAlignComment,RTrim,SettersAndGettersPass|golang,ConstructorPass|golang,OrderMethod,NormalizeLnAndLtrimLines,Reindent

class A {
	private $a;
	public $b;
	protected $c;
	private function A(){
		return $this->a;
	}
	public function B(){
		return $this->b;
	}
	protected function C(){
		return $this->c;
	}
	private function SetA($a){
		$this->a = $a;
	}
	public function SetB($b){
		$this->b = $b;
	}
	protected function SetC($c){
		$this->c = $c;
	}
	function __construct($a, $b, $c){
		$this->SetA($a);
		$this->SetB($b);
		$this->SetC($c);
	}
}
